src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -302,6 +305,65 @@ private String get_user_myworkspace_site_json(HttpServletRequest request) {
 	}
 
 	/**
+	 * retain only the site record where the current user has Owner role inside
+	 * @param request
+	 * @param projectsString
+	 * @param currentUserId
+	 * @return
+	 */
+	private String filterForSitesWithOwnerRole(HttpServletRequest request,
+			String projectsString, String currentUserId) {
+		JSONArray ownerSitesJSONArray = new JSONArray();
+		try {
+			JSONObject sitesJSONObject = new JSONObject(projectsString);
```

Comment:
What does projectsString contain?  Is it a string of JSON?  It seems like in the calling code, it was a plain string, then encoded in a string of JSON.  Maybe it should have been a JSON object in the calling code and that object passed into this method.  It's better to have less conversion back and forth between object and string.